notebooks/niriss_ami_binary/3_niriss_ami_binary.ipynb

```diff
@@ -0,0 +1,645 @@
+{
```

Comment:
<p>From an earlier cell, it looks like <code>wavel</code> (defined as<code>hdu['OI_WAVELENGTH'].data['EFF_WAVE']</code>)  is only length 1 for me. That may cause the error I see here. Do you get a different result?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jdat_notebooks/pull/100/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
For now, this is only fixed by going back to commit 5d1a379 of CANDID. To avoid the error, we may have to pin to that commit unless we see a response to amerand/CANDID#1.